frontend/ic_vetkeys/src/utils/utils.ts

```diff
@@ -306,6 +306,34 @@ export function augmentedHashToG1(
     return pt;
 }
 
+/**
+ * Verify a BLS signature
+ *
+ * A VetKey is in the end a valid BLS signature; this function checks that a
+ * provided BLS signature is the valid one for the provided public key and
+ * message.
+ *
+ * When a VetKey struct is created (VetKey.decryptAndVerify) the signature (aka
```

Comment:
IIRC, `decryptAndVerify` lives on `EncryptedVetKey` (not `VetKey`)

Reply:
Fixed thanks